src/library/Uno.Material/Styles/Controls/v2/CalendarDatePicker.xaml

```diff
@@ -183,7 +187,7 @@
 						</FlyoutBase.AttachedFlyout>
 
 						<Border x:Name="Background"
-								Background="{StaticResource OnSurfaceBrush}"
+								Background="{StaticResource M3MaterialCalendarDatePickerBackground}"
```

Comment:
how come `OnSurfaceBrush` didnt resolve?

Reply:
It was resolving but onsurface was just the wrong color to use. Surface is pretty much all white or all black depending on the theme